FarmSimulator1/src/Domain/AnimalProduct.java

```diff
@@ -0,0 +1,30 @@
+package Domain;
```

Comment:
all your classes should have packages, including Main
package naming convention is with lower case starting letter, so this would be "domain"

It's a good practice to group your packages under a common root so I would suggest having:

ro.cbanu.domain

Or if you want to keep this homework separate

ro.cbanu.oop.domain -> and then your Main could be in ro.cbanu.oop for example